projects/robots/gctronic/e-puck/plugins/robot_windows/botstudio/e-puck/EPuckSlider.cpp

```diff
@@ -186,6 +192,24 @@ EPuckSlider::~EPuckSlider() {
   delete mTextItem;
 }
 
+void EPuckSlider::sliderPressed() {
+  mWasDragged = false;
+}
+
+void EPuckSlider::sliderMoved(int value) {
+  mWasDragged = true;
+  mPreviousValue = value;
+}
+
+void EPuckSlider::sliderReleased() {
+  if (mType == RevertibleSlider && !mWasDragged) {
+    const int v = mSlider->value();
+    if (v == mPreviousValue)
+      setInverted(!isInverted());
```

Comment:
Would not be better to also call `update()` here?

Reply:
I just added it, but it doesn't change anything to the behavior...